src/Samples/NetCoreClient/Program.cs

```diff
@@ -1,30 +1,191 @@
 ﻿using System;
-using StandardClient;
-using CoreWCF.Samples.StandardCommon;
+//using StandardClient;
+//using CoreWCF.Samples.StandardCommon;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Threading.Tasks;
+using Contract;
 
 namespace NetCoreClient
 {
-    class Program
+    public class Program
     {
         /// <remarks>
-        /// use commanline argument localhost
+        /// use commanline argument "http://localhost" BasicHttpBinding
         /// or something similar to indicate the WCF Server hostname
         /// </remarks>
         static void Main(string[] args)
         {
-            string hostname = args.Length >= 1 ? args[0] : null;
+            string hostAddr = args.Length >= 1 ? args[0] : "http://localhost:8088";
 
             Console.Title = "WCF .Net Core Client";
-            Settings settings = ClientLogic.BuildClientSettings(hostname);
+            PrintScenarioList();
 
-            static void log(string value) => Console.WriteLine(value);
-            ClientLogic.InvokeEchoServiceUsingWcf(settings, log);
+            while (true)
+            {
+                Console.WriteLine("Type scenario number, ? for scenario list or Hit enter to exit");
```

Comment:
@mconnew, @g7ed6e  The previous form of this sample was to me a bit overly convoluted. Does it make it more readable (and directly stealable from) to break it up into self contained scenarios?
Should each of them be split into a file per binding?

Reply:
i think samples should :
- contain only the required code to required for its purpose
- allow user to copy/paste and get started

i would go with several folders each containing a working project.